packages/react-router-dom/rollup.config.js

```diff
@@ -23,7 +23,7 @@ const babelOptionsESM = {
 const commonjsOptions = {
   include: /node_modules/,
   namedExports: {
-    "../react-router/node_modules/react-is/index.js": ["isValidElementType"]
+    "react-is": ["isValidElementType"]
```

Comment:
This change is breaking the build. Is it necessary? Do we need to bump react-is?

Reply:
Hm.. weird. This is broken for me locally.

Reply:
Reverted